nats/build.gradle

```diff
@@ -6,6 +6,7 @@ dependencies {
     api "io.micronaut:micronaut-inject-java"
     api "io.nats:jnats:$natsVersion"
 
+    compileOnly "jakarta.inject:jakarta.inject-api:2.0.0"
```

Comment:
This is not necessary because it's already included in Micronaut 3.0

Reply:
I've removed the dependency